app/src/main/java/com.myfitnesstracker.dao/IBMI_DAO.kt

```diff
@@ -0,0 +1,2 @@
+interface IBMI_DAO {
```

Comment:
Use UpperCamelCase for class and interface names, and avoid using the underscore _